docs/topics/extensions.rst

```diff
@@ -66,7 +66,7 @@ Disabling an extension
 
 In order to disable an extension that comes enabled by default (ie. those
 included in the :setting:`EXTENSIONS_BASE` setting) you must set its order to
-``None``. For example::
+``None``(or ``0``). For example::
```

Comment:
hey @darshanime! Setting `0` won't disable an extension. Compare how [all default extensions have a value of `0`](http://doc.scrapy.org/en/latest/topics/settings.html#extensions-base), and these are all enabled. ;)